src/coreclr/vm/threads.h

```diff
@@ -1395,9 +1395,8 @@ class Thread
         WRAPPER_NO_CONTRACT;
 #ifdef PROFILING_SUPPORTED
         _ASSERTE(PreemptiveGCDisabled()
-                 || CORProfilerPresent() ||    // This added to allow profiler to use GetILToNativeMapping
+                 || CORProfilerPresent());  // This added to allow profiler to use GetILToNativeMapping
```

Comment:
should the comment be updated?

Reply:
I have deleted check related to shutdown.

This comment talks about steady state. I think it is part of the workaround described in https://github.com/dotnet/runtime/blob/f124c0efbbd527b730297c49758967ec110e35fe/src/coreclr/vm/proftoeeinterfaceimpl.cpp#L4832-L4836